### DISCUSSION TITLE
Gradient color bug

### DISCUSSION THREAD

**Discussion by bbhxwl** [AvaloniaUI/Avalonia#6398] [Q&A]
Avalonia
![image](https://user-images.githubusercontent.com/26525599/128841233-26f26585-d195-441f-a90c-892d193de2af.png)

wpf 
![image](https://user-images.githubusercontent.com/26525599/128841285-20db3a03-3bb6-4966-8423-df37c8464581.png)

**Comment by Gillibald** [comment]
` <LinearGradientBrush StartPoint="0%,0%" EndPoint="100%,100%">`
